src/components/BreadCrumbs.vue

```diff
@@ -10,20 +23,115 @@
 </template>
 
 <script setup lang="ts">
-import { reactive } from "vue";
+import { reactive, Reactive, watch } from "vue";
+import { useRoute } from "vue-router";
+import { ROUTES } from "@/constants/router";
+import IBreadCrumbsItem from "@/types/IBreadCrumbsItem";
+import {
+  SECTIONS_NAMES,
+  SECTIONS_ICONS,
+  TABS_NAMES,
+} from "@/constants/breadCrumbs";
+import { useUsersStore } from "@/store/users";
+import { useCVsStore } from "@/store/cvs";
 
-const items = reactive([
-  {
-    title: "Home",
+const breadcrumbsItems: Reactive<IBreadCrumbsItem[]> = reactive([]);
+
+const route = useRoute();
+
+watch(route, updateBreadCrumbs);
+
+updateBreadCrumbs();
+
+function updateBreadCrumbs() {
+  const [section, id, tab] = route.fullPath.slice(1).split("/");
+
+  breadcrumbsItems.splice(0, breadcrumbsItems.length);
+
+  breadcrumbsItems.push({
+    crumbNum: 1,
+    title: SECTIONS_NAMES.HOME,
     disabled: false,
-    href: "breadcrumbs_home",
-  },
-  {
-    title: "Employees",
-    disabled: true,
-    href: "breadcrumbs_employees",
-  },
-]);
+    to: {
+      path: ROUTES.USERS.PATH,
+    },
+    crumbIconName: SECTIONS_ICONS.HOME,
+  });
+
+  if (section) {
+    breadcrumbsItems.push({
+      crumbNum: 2,
+      title: SECTIONS_NAMES[section],
+      disabled: id === undefined,
+      to: {
+        path: `/${section}`,
+      },
+    });
+  }
+
+  if (id) {
+    breadcrumbsItems.push({
+      crumbNum: 3,
+      title: "Loading...",
+      disabled: tab === undefined,
+      to: {
+        path: `/${section}/${id}`,
+      },
+      crumbIconName: SECTIONS_ICONS[section],
+    });
+
+    const { getUserById } = useUsersStore();
+    const { getCVById } = useCVsStore();
+
+    switch (section) {
+      case ROUTES.USERS.PATH.slice(1):
+        getUserById(Number(id))
+          .then((userData) => {
+            if (!userData) throw new Error("Empty user data!");
+            breadcrumbsItems[2].title = `${userData?.firstName} ${userData?.lastName}`;
+          })
+          .catch(() => {
+            breadcrumbsItems[2].title = `❌ Loading Error`;
+          });
+        break;
+      case ROUTES.CVS.PATH.slice(1):
+        getCVById(Number(id))
+          .then((cvData) => {
+            if (!cvData) throw new Error("Empty CV data!");
+            breadcrumbsItems[2].title = `${cvData?.name}`;
+          })
+          .catch(() => {
+            breadcrumbsItems[2].title = `❌ Loading Error`;
```

Comment:
We should hide tabs if there is no user with id given in the route full path and add a link to main page.